README.md

```diff
@@ -79,17 +79,19 @@ overview of all settings configurable through the CLI.
-| `Worker.RemotePassword`               | API password for the remote workers                 | -                                | -                                | `RENTERD_WORKER_API_PASSWORD`                     | `worker.remotes`              |
 | `Autopilot.Enabled`					| Enables/disables autopilot							| `true`							| `--autopilot.enabled`			| `RENTERD_AUTOPILOT_ENABLED`						| `autopilot.enabled`					|
 | `Autopilot.AccountsRefillInterval`   | Interval for refilling workers' account balances     | `24h`                             | `--autopilot.accountRefillInterval` | -                                              | `autopilot.accountsRefillInterval`  |
 | `Autopilot.Heartbeat`                | Interval for autopilot loop execution                | `30m`                             | `--autopilot.heartbeat`            | -                                              | `autopilot.heartbeat`               |
-| `Autopilot.MigrationHealthCutoff`    | Threshold for migrating slabs based on health        | `0.75`                            | `--autopilot.migrationHealthCutoff` | -                                              | `autopilot.migrationHealthCutoff`   |
 | `Autopilot.RevisionBroadcastInterval`| Interval for broadcasting contract revisions         | `168h` (7 days)                   | `--autopilot.revisionBroadcastInterval` | `RENTERD_AUTOPILOT_REVISION_BROADCAST_INTERVAL` | `autopilot.revisionBroadcastInterval` |
 | `Autopilot.ScannerBatchSize`         | Batch size for host scanning                         | `1000`                            | `--autopilot.scannerBatchSize`      | -                                              | `autopilot.scannerBatchSize`        |
 | `Autopilot.ScannerInterval`          | Interval for scanning hosts                          | `24h`                             | `--autopilot.scannerInterval`       | -                                              | `autopilot.scannerInterval`         |
 | `Autopilot.ScannerNumThreads`        | Number of threads for scanning hosts                 | `100`                             | -                                | -                                              | `autopilot.scannerNumThreads`       |
-| `Autopilot.MigratorParallelSlabsPerWorker` | Parallel slab migrations per worker                    | `1`                               | `--autopilot.migratorParallelSlabsPerWorker` | `RENTERD_MIGRATOR_PARALLEL_SLABS_PER_WORKER` | `autopilot.migratorParallelSlabsPerWorker` |
+| `Migrator.HealthCutoff`              | Threshold for migrating slabs based on health        | `0.75`                            | `--migrator.healthCutoff` | -                                              | `migrator.HealthCutoff`   |
+| `Migrator.ParallelSlabsPerWorker`    | Parallel slab migrations                             | `1`                               | `--migrator.parallelSlabsPerWorker` | `RENTERD_MIGRATOR_PARALLEL_SLABS_PER_WORKER` | `migrator.parallelSlabsPerWorker` |
```

Comment:
This one we can keep but I'd rename it since there is no more workers involved. I'm not sure the `Migrator` prefix makes sense though. 

I think I also prefer adding a section to the autopilot rather than adding a section in the root. To be more consistent with what we do for scanning and because the migrator isn't its own root component. 

We probably could clean things up here a bit more anyway (but not this PR) since S3 should now also be housed under Worker rather than be its own thing.

Reply:
Ok, I'll house it under the autopilot config. Perhaps `Autopilot.MigratorNumThreads` is a good name. If we were to do that I would keep the `migrator` prefix to be consistent with the other config.